app/src/pages/trace/NewSpanAnnotationForm.tsx

```diff
@@ -12,15 +15,30 @@ export type NewSpanAnnotationFormProps = {
 
 export function NewSpanAnnotationForm(props: NewSpanAnnotationFormProps) {
   const { annotationName: name, spanNodeId, onCreated } = props;
+  const { projectId } = useParams();
+  const { timeRange } = useLastNTimeRange();
```

Comment:
we can't assume this UI is nested under a project. this UI is also visible under experiments. I think we just let the auto-refresh take care of things.

Reply:
auto refresh as in the streaming? i can remove these spreads then, i wasn't seeing the annotations bar update though but i had streaming off

Reply:
removed!